src/Behavioral.Automation/Bindings/ClickBinding.cs

```diff
@@ -27,10 +38,18 @@ public void ClickTwice([NotNull] IWebElementWrapper element)
         public void ClickThreeTimes([NotNull] IWebElementWrapper element)
         {
             element.Click();
+            _debugBinding.Wait(1);
```

Comment:
What do you think about adding ability to define wait interval. Something like "user clicked two times on "element" with 1 second interval? Just a suggestion

Reply:
not sure if we really need this. User usually clicks on elements not immediately and this Wait construction is just to avoid double-click behavior.